Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -341,9 +341,8 @@ void CFrame::DoOpen(bool Boot)
   wxString path = wxFileSelector(
       _("Select the file to load"), wxEmptyString, wxEmptyString, wxEmptyString,
       _("All GC/Wii files (elf, dol, gcm, iso, tgc, wbfs, ciso, gcz, wad)") +
-          wxString::Format(
-              "|*.elf;*.dol;*.gcm;*.iso;*.tgc;*.wbfs;*.ciso;*.gcz;*.wad;*.dff;*.tmd|%s",
-              wxGetTranslation(wxALL_FILES)),
+          wxString::Format("|*.elf;*.dol;*.gcm;*.iso;*.tgc;*.wbfs;*.ciso;*.gcz;*.wad;*.dff|%s",
```

Comment:
Huh, interesting undocumented behaviour. ;) Seems to be due to NANDContentLoader accepting two types of paths, the title content dir and the TMD. Unfortunately, the way it works is kind of incompatible with future cleanups.

I plan to re-add a proper way to boot NAND titles soon.